src/utils/misc.ml

```diff
@@ -881,6 +881,14 @@ module String = struct
       let equal (x : string) (y : string) : bool = (x = y)
       let hash = Hashtbl.hash
     end)
+
+  let split_once str ~idx =
+    let n = length str in
+    sub str 0 idx, sub str (idx + 1) (n - idx - 1)
+
+  let split_first_exn str ~split_on =
```

Comment:
Maybe remove this stuff if we don't need it? But keeping it isn't so bad either.

Reply:
Oops thanks for the catch. I added this in before we decided not to support `-libloc`, which removed the need for this.